spec/models/order_spec.rb

```diff
@@ -1,6 +1,33 @@
 require "rails_helper"
 
 RSpec.describe Order, type: :model do
+  context "valid attributes" do
+
+  let!(:loan_request) { LoanRequest.create(title: "Farm Tools",
+                                           description: "help out with the farm tools",
+                                           amount: "100",
+                                           requested_by_date: "2015-06-01",
+                                           repayment_begin_date: "2015-12-01",
+                                           contributed: "50",
+                                           repayment_rate: "monthly",
+                                           user_id: borrower.id)
+  }
+    let(:borrower) { User.create(email: "example@example.com", password: "password", name: "sally", role: 1) }
+    let(:order) { Order.create(user_id: 1, cart_items: {"#{loan_request.id}" => 25})}
```

Comment:
Line is too long. [85/80]<br>Inconsistent indentation detected.<br>Space missing inside }.<br>Space inside { missing.<br>Space inside } missing.